packages/ibm-products/src/components/Card/Card.js

```diff
@@ -77,6 +84,7 @@ export let Card = forwardRef(
     },
     ref
   ) => {
+    getStarted ? (actionIcons = metadata) : '';
```

Comment:
i haven't come across this notation before. it's very interesting. personally i'm not a fan of mutating existing variables like this. _if_ we need this new `metadata` prop and we can't just use `actionIcons` i would rather see a getter function used here instead to instantiate the variable that contains the icons. something like `const getIcons = () => getStarted ? metadata : actionIcons`